blobs/howto/JavaScript/blob-query-endpoint/index.js

```diff
@@ -0,0 +1,58 @@
+const { ResourceManagementClient } = require("@azure/arm-resources");
+const { StorageManagementClient } = require("@azure/arm-storage");
+const { DefaultAzureCredential } = require("@azure/identity");
+const { BlobServiceClient } = require("@azure/storage-blob");
+
+// <Snippet_query_blob_endpoint>
+async function getBlobServiceEndpoint(saName, credential) {
+  const subscriptionId = "<subscription-id>";
+  const rgName = "<resource-group-name>";
+
+  const resourceMgmtClient = new ResourceManagementClient(
+    credential,
+    subscriptionId
+  );
+
+  // Register the Storage resource provider in the subscription
+  resourceMgmtClient.providers.register("Microsoft.Storage");
```

Comment:
Do we need to register `Microsoft.Storage` every time? If no please comment this out.

Reply:
Good point. I think I'll pull this piece of the code out of the main sample and add some detail in the corresponding article. Thanks for bringing this up!